.github/workflows/container.yaml

```diff
@@ -0,0 +1,65 @@
+name: Container build & push jobs
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+*"
```

Comment:
Github's tag filtering capabilities are not using regex. So I use a simpler filter here to take anything of the shape `vX.Y.Z*` and later before pushing the release image the full regex from semver.org is applied and verified.